node/pkg/watchers/solana/shim.go

```diff
@@ -226,6 +230,10 @@ func (s *SolanaWatcher) shimProcessInnerInstruction(
 	alreadyProcessed ShimAlreadyProcessed,
 	isReobservation bool,
 ) (bool, error) {
+	if startIdx >= len(innerInstructions) {
+		return false, fmt.Errorf("startIdx %d is greater than or equal to %d", startIdx, len(innerInstructions))
```

Comment:
```suggestion
		return false, fmt.Errorf("startIdx %d is is out of bounds of slice innerInstructions (length: %d)", startIdx, len(innerInstructions))
```
From the error log it's not clear why e.g. "startIdx 3 is greater than or equal to 3" is a problem